core/modules/qdisp/QueryResource.h

```diff
@@ -58,7 +58,7 @@ class QueryResource : public XrdSsiService::Resource {
                   ExecStatus& status)
         : Resource(rPath.c_str()),
           _session(NULL),
-          _request(NULL),
+          _request(),
```

Comment:
I agree, I missed that, I am not sure what Daniel was thinking, but I'll fix it.